src/common/feedGenerator.ts

```diff
@@ -538,6 +539,16 @@ export const sourceFeedBuilder = (
 
   if (sourceId === 'community') {
```

Comment:
So for community we ignore vordr?

Reply:
Probably not 🙈 
Forgot this part I think. Will update query to work for community as well

Reply:
Fixed so that it works on community as well. Since we already hide banned posts from community picks, I moved the andWhere statement.